circuit_puzzles/include/statutes_utils.clib

```diff
@@ -46,15 +46,17 @@
   (defconst STATUTE_ANNOUNCER_DISAPPROVAL_COOLDOWN_INTERVAL 36)
   (defconst STATUTE_MIN_BID_INCREASE_BPS 37)
 
-  (defconst STATUTES_ANN_PREFIX 0x54) ; chr(84) -> "T" for sTaTuTes
+  ;(defconst STATUTES_ANN_PREFIX 0x54) ; chr(84) -> "T" for sTaTuTes
   (defconst STATUTES_MUTATION_PREFIX "m")
   (defconst STATUTE_PREFIX "s")
   (defconst STATUTE_FULL_PREFIX "S")
   (defconst PRICE_PREFIX "p")
+  (defconst PROTOCOL_PREFIX "CRT")
```

Comment:
still feels like a waste to use 3 bytes for prefix

Reply:
shortening it to 'C'